api/db/populate_users.js

```diff
@@ -0,0 +1,121 @@
+#! /usr/bin/env node
+
+// TODO: This will all be in the general js to abide by DRY principal
+const userArgs = process.argv.slice(2);
+
+if (!userArgs[0].startsWith('mongodb')) {
+  console.log(
+    'Error: you need to specify a valid MongoDB URL as the first argument'
+  );
+
+  return;
+}
+
+const User = require('../models/User');
+const Interest = require('../models/Interest')
+const Token = require('../models/Token')
+const crypto = require('crypto');
+
+const async = require('async');
+
+const mongoose = require('mongoose');
+const mongoDB = userArgs[0];
+mongoose.connect(mongoDB, { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.Promise = global.Promise;
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'MongoDB connection error:'));
+
+const { uniqueNamesGenerator, names } = require('unique-names-generator');
+const countries = require('country-region-data');
+const locales = ['en', 'es'];
+const users = [];
+
+const createUser = async (
+  name, email, emailConfirmed, emailConfirmedAt,
+    dateOfBirth, country, region,
+    preferredLocale, interests, admin,
+    callback
+) => {
+  const user = new User({
+    name, email, emailConfirmed, emailConfirmedAt,
+    dateOfBirth, country, region,
+    preferredLocale, interests, admin
+  });
+  user.setPassword('Password1');
+  await user.save();
+  const token = createTokenForUser(user);
+  await token.save();
+
+  console.log(`User: ${user}`);
```

Comment:
```suggestion
  console.log(`New user: ${user}`);
```

Nitpick: for consistency with the interests script.

Reply:
Works for me. This has been resolved.